ams/api/src/main/java/com/netease/arctic/ams/api/client/ArcticThriftUrl.java

```diff
@@ -223,13 +237,23 @@ public String serverUrl() {
 
   @Override
   public String toString() {
-    return "ArcticThriftUrl{" +
-        "schema='" + schema + '\'' +
-        ", host='" + host + '\'' +
-        ", port=" + port +
-        ", catalogName='" + catalogName + '\'' +
-        ", socketTimeout=" + socketTimeout +
-        ", url='" + url + '\'' +
-        '}';
+    return "ArcticThriftUrl{"
```

Comment:
How about changing the implementation with `MoreObjects`?

Reply:
This could be in an another PR.

Reply:
fixed.